modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/DataLayoutBuilder.es.js

```diff
@@ -52,13 +57,8 @@ class DataLayoutBuilder extends Component {
 	}
 
 	disposed() {
-		if (this._translationManager) {
-			this._translationManager.detach(
-				'availableLocales',
-				this.onAvailableLocalesRemoved.bind(this)
-			);
-
-			this._translationManager = null;
+		if (this._translationManagerHandles) {
+			this._translationManagerHandles.forEach(handle => handle.detach());
```

Comment:
The common pattern is that `.on` returns an event handle that can be detached

Reply:
Looks fine!

Reply:
Makes sense.